dail/postponed-expedited-snap-verifications.vbs

```diff
@@ -69,33 +70,143 @@ fifteen_of_appl_month = left(SNAP_application_date, 2) & "/15/" & right(SNAP_app
+					next_line = trim(next_line)
+				Loop until next_line = "---"                                    'stop at the dividing line
+				'if there were lines saved
+				If verif_note_lines <> "" Then
+					verif_counter = 1                                           'setting a counter to find verifs that have been numbered
+					If left(verif_note_lines, 3) = "~|~" Then verif_note_lines = right(verif_note_lines, len(verif_note_lines) - 3)             'making an array of all of the lines
+					If InStr(verif_note_lines, "~|~") = 0 Then
+						verif_lines_array = Array(verif_note_lines)
+					Else
+						verif_lines_array = split(verif_note_lines, "~|~")
+					End If
+
+					verifs_to_add = ""                                          'blanking a string for adding all the lines together
+					For each line in verif_lines_array
+						counter_string = verif_counter & ". "                   'using the counter - which is a number to make a string that looks like what is in the note
+						If left(line, 2) = "- " OR left(line, 3) = counter_string Then                          'If the string starts with a dash or the counter
+							If left(line, 2) = "- " Then line = "; " & right(line, len(line) - 2)               'Removes the list delimiter and adds the editbox delimiter
+							If left(line, 3) = counter_string Then line = "; " & right(line, len(line) - 3)
+							verif_counter = verif_counter + 1                                                   'incrementing the counter
+						Else
+							line = " " & line                                                                   'adding a space to the sting so there is a space between words if we are at a 'same line'
+						End If
+
+						verifs_to_add = verifs_to_add & line                    'adding the verif information all together
+					Next
+					If left(verifs_to_add, 2) = "; " Then verifs_to_add = right(verifs_to_add, len(verifs_to_add) - 2)  'trimming the string
+					If verifs_to_add <> "" Then verifs_needed = trim(verifs_needed & verifs_to_add) 'adding the information to the variable used in this script 
+				End If
+			End If
+			PF3         'leaving the note
+		Else
+			If note_list_header <> "First line of Case note" Then PF3           'this backs us out of the note if we ended up in the wrong note.
+		End If
+	End If
+
+	if note_date = "        " then Exit Do                                      'if we are at the end of the list of notes - we can't read any more
+
+	note_row = note_row + 1
+	if note_row = 19 then
+		note_row = 5
+		PF8
+		EMReadScreen check_for_last_page, 9, 24, 14
+		If check_for_last_page = "LAST PAGE" Then Exit Do
+	End If
+	EMReadScreen next_note_date, 8, note_row, 6
+	if next_note_date = "        " then Exit Do
+Loop until DateDiff("d", too_old_date, next_note_date) <= 0
+
+'Navigate back to DAIL
+PF3
+
+'Navigates to SPEC/MEMO through DAIL to maintain tie to list
```

Comment:
I moved the navigation and creation of the SPEC/MEMO to after the dialog. However, I am unsure about the navigation handling if a worker were to leave the DAIL screen to find case info. I could use navigate_to_MAXIS_screen but then we would lose the connection to the DAIL. Since we wouldn't necessarily know what panel they navigated to, should I add handling to navigate back to DAIL and find the same DAIL message after using the navigate_to_MAXIS_screen function? Or am I overthinking this?